pkg/lib/cockpit-components-password.jsx

```diff
@@ -131,8 +168,20 @@ export const PasswordFormFields = ({
             {password_confirm_label && <FormGroup label={password_confirm_label}
                        id={idPrefix + "-pw2-group"}
                        fieldId={idPrefix + "-pw2"}>
-                <TextInput type="password" id={idPrefix + "-pw2"} autoComplete="new-password"
-                           value={passwordConfirm} onChange={(_event, value) => { setConfirmPassword(value); change("password_confirm", value) }} />
+                <InputGroup>
+                    <InputGroupItem isFill>
+                        <TextInput type={passwordConfirmHidden ? "password" : "text"} id={idPrefix + "-pw2"} autoComplete="new-password"
+                            value={passwordConfirm} onChange={(_event, value) => { setConfirmPassword(value); change("password_confirm", value) }} />
+                    </InputGroupItem>
+                    <InputGroupItem>
+                        <Button
+                            variant="control"
+                            onClick={() => setPasswordConfirmHidden(!passwordConfirmHidden)}
+                            aria-label={passwordConfirmHidden ? _("Show confirmation password") : _("Hide confirmation password")}>
+                            {passwordConfirmHidden ? <EyeIcon /> : <EyeSlashIcon />}
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19220-20230822-125449-71e7c813-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)